rust-bot/src/main.rs

```diff
@@ -127,10 +162,11 @@ fn main() {
     // Shards will automatically attempt to reconnect, and will perform
     // exponential backoff until it reconnects.
     if let Err(why) = client.start() {
+        error!(?why);
```

Comment:
👀  I have not seen ? prefix before. What does it do? is it error-macro related?

Reply:
Yeah. The `?` Prefix (called a sigil in Rust?) Records it using the fmt::Debug implementation 

There's also the `%` sigil which uses the fmt::Display implementation. 

Docs here: https://docs.rs/tracing/0.1.19/tracing/#recording-fields

Reply:
TIL

Reply:
I should probably clarify that this is only part of the tracing macros afaict. So it won't work just anywhere.